askama_parser/src/lib.rs

```diff
@@ -14,21 +14,18 @@ use nom::multi::separated_list1;
 use nom::sequence::{delimited, pair, tuple};
 use nom::{error_position, AsChar, IResult, InputTakeAtPosition};
 
-pub use self::expr::Expr;
-pub use self::node::{
-    BlockDef, Call, Cond, CondTest, Extends, If, Import, Include, Let, Lit, Loop, Macro, Match,
-    Node, Raw, Target, When, Whitespace, Ws,
-};
-
-mod expr;
-mod node;
+use self::node::Node;
```

Comment:
This should be spelled as `use node::Node;`, and please move it directly underneath the `pub mod node;` declaration.

I'm inclined to `pub use node::Node;` and `pub use expr::Expr;` which are arguably higher-level types, and leave the inner types as things that should be imported through the module name, what do you think?